servicer_test.go

```
@@ -105,7 +105,7 @@ func TestService_Get(t *testing.T) {
 					Zone:       &location,
 				},
 			}, nil
-		})
+		}).AnyTimes()
```

Please don't ignore errors while we aren't finding the root cause.